config/routes.rb

```diff
@@ -1,2 +1,4 @@
 Rails.application.routes.draw do
+  get ":slug/next", to: "redirections#next"
```

Comment:
What are the advantages to having a custom slug versus using the referrer and `/next`?